common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/rest/RestClientVrops.java

```diff
@@ -441,14 +536,61 @@ public PolicyDTO.Policy getPolicyContent(PolicyDTO.Policy policy) {
 		}
 
 		if (!HttpStatus.OK.equals(response.getStatusCode())) {
-			throw new RuntimeException(
-					String.format("Error exporting all policy %s from vROPS : remote REST service returned: %s", policy.getName(), response.getStatusCode()));
+			throw new RuntimeException(String.format("Error exporting all policy %s from vROPS : remote REST service returned: %s", policy.getName(), response.getStatusCode()));
```

Comment:
It may be a good idea to include response body in exception details. Response from server may have additional details what is wrong with the request and its body.

Reply:
Response body included.